### DISCUSSION TITLE
Is there a way to get the correct version number in Mixplorer Beta?

### DISCUSSION THREAD

**Discussion by Achyuth072** [ImranR98/Obtainium#1904] [Q&A] [answered]
I added Mixplorer Beta through obtainum apps but its displays a string of numbers as installed version rather than the correct version whixh would be 6.68.0-Beta. Is this behavior due to the download link being a website? 

https://mixplorer.com/beta/

**Comment by DarkCrypt** [comment, marked as answer]
See issue #1786.

**Reply by Achyuth072** [reply]
I guess for now it's a temporary solution.

**Reply by DarkCrypt** [reply]
Temporary? For version extraction?

**Reply by Achyuth072** [reply]
Wait does your config also fix the x64 issue as well? I saw your last reply to that issue so I assumed that couldn't be fixed.

**Reply by DarkCrypt** [reply]
My last comment in that issue was about the app config website. My comment directly above it is my fix. The install link I posted is for arm64 but the config breakdown also shows all versions.

**Reply by Achyuth072** [reply]
My bad then!
